src/main/resources/application.yml

```diff
@@ -59,6 +59,7 @@ spring:
     username: actionsvc
     password: actionsvc
     driverClassName: org.postgresql.Driver
+    continue-on-error: true
```

Comment:
Is this something we want in all environments? Would it not be better to add this in our ras-rm-docker-dev configuration instead?

Reply:
I think it only applies to the local environment because it's not been added in the application-cloud.yml